Sources/BSplineLib/ParameterSpaces/parameter_space.inl

```diff
@@ -416,10 +416,10 @@ ParameterSpace<para_dim>::InsertKnot(Dimension const& dimension,
   }
 #endif
   KnotVector& knot_vector = *knot_vectors_[dimension_value];
-  Multiplicity const insertion{
-      std::min(multiplicity.Get(),
-               degrees_[dimension_value]
-                   - knot_vector.DetermineMultiplicity(knot, tolerance).Get())};
+  Multiplicity const insertion{std::min(
```

Comment:
```suggestion
  // instead of (degree - current_multiplicity),
  // clip at (degree - current_multiplicity) + 1 is to allow C^-1
  Multiplicity const insertion{std::min(
``` 
does this make sense? @jzwar

Reply:
Yes! IMO this is perfectly fine!